scheduled_tasks/poll_injection_responses_task.rb

```diff
@@ -6,7 +6,7 @@ class PollInjectionResponsesTask < Scheduler::SchedulerTask
   every '1m'
 
   def run
-    queue = Settings.aws.response_queue
+    queue = Settings.aws.sqs.response_queue_url || Settings.aws.response_queue
```

Comment:
just a query. is the `response_queue_url` fake value from settings going to be nilled, or should it not be set to an ENV var value so that this code falls back to  the `Settings.aws.response_queue` for environments that do not have the env var

Reply:
It will be nil unless we explicitly set one.  The setting gem looks for an env var called SETTINGS__AWS__SQS__RESPONSE_QUEUE_URL and loads it from there... I'd expect that to be nil by default, but I want to check on demo before I fully deploy!

Reply:
always forget about that settings gem. another magic abstraction.